### PR TITLE
feat: Backlog/connect utils cleanup

### PR DESCRIPTION
**Pull request by lluisFtrack**
Resolves : 

* CLICKUP-
* FT-
* SENTRY-
* ZENDESK-

- [ ] I have added automatic tests where applicable.
- [ ] The PR contains a description of what has been changed.
- [ ] The description contains manual test instructions.
- [ ] The PR contains update to the release notes.
- [ ] The PR contains update to the documentation.

This PR has been tested on :

- [ ] Windows.
- [ ] MacOs.
- [ ] Linux.

## Changes
- Bit of clean up sorting all the connect utils in the utils folder

### CODE REVIEW

File: apps/connect/source/ftrack_connect/ui/application.py [hunk 1]
```diff
@@ -29,13 +29,17 @@
...
     get_plugin_data,
-    open_directory,
     PLUGIN_DIRECTORIES,
 )
+from ftrack_connect.utils.login import (
+    read_json_config,
+    write_json_config,
+)
+from ftrack_connect.utils.directory import open_directory
 import ftrack_connect.ui.theme
```
Comment (henriknorin-ftrack) [inline apps/connect/source/ftrack_connect/ui/application.py:43]: Perhaps change all these to the form `from ftrack_connect.ui import theme` and then use `theme.` to align with rest of code base.

**Review by henriknorin-ftrack — APPROVED** [review]
LGTM, with one suggestion.

Question: how much can be moved to/reused from utils library?

File: apps/connect/source/ftrack_connect/ui/application.py (hunk 1 above)
Comment (lluisFtrack) [reply apps/connect/source/ftrack_connect/ui/application.py:43]: I think we should keep that for another task, we will never end if we try to clean the connect code to much.

**Review by henriknorin-ftrack — APPROVED** [review]
LGTM, tested with two suggestions

**Review by henriknorin-ftrack — APPROVED** [review]

**Review by henriknorin-ftrack — APPROVED** [review]

**Pull request merged**